wordpress-mobile/WordPress-iOS
# Merge Media Editing branch into develop

Adds Media Editing capabilities.

### To test

1. In Aztec, tap an image
2. Tap "Edit"
3. Make sure the Media Editor opens

All the other changes were tested in the PRs submitted to this feature branch.

PR submission checklist:
